.github/workflows/ci-cd.yml

```diff
@@ -52,6 +38,17 @@ jobs:
         working-directory: ./ui
         run: npm run prettier
 
+  typocheck:
+    name: Spell Check with Typos
```

Comment:
This name doesn't match the previous ones. If we want it to be the same (idk if we do), then it should be `runner / typos` ?

Reply:
Yeah good catch, I'll update that